roles/openshift_preflight/nodes/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+# determine if yum install of node pkgs will work
+- when:
+    - not containerized
+  block:
+
+    - name: main packages for enterprise
+      when:
+        - deployment_type == "openshift-enterprise"
+      check_yum_update:
+        packages:
+          - atomic-openshift
+          - atomic-openshift-node
+          - atomic-openshift-sdn-ovs
```

Comment:
Same comment here about using a variable for the package name prefix to avoid having separate tasks for each deployment_type.